src/CurrentBook.java

```diff
@@ -8,25 +8,31 @@ public class CurrentBook extends Book{
     private int groupID;
     private String bookBuddy;
     private boolean hasOpenSearch;
+    //NEW SPRINT3
```

Comment:
new attribute to track if the user finished the book (only relevant when in a book group for the book)